ocs_sample_library_preview/Roles.py

```diff
@@ -0,0 +1,105 @@
+from .BaseClient import BaseClient
+from .Security.Role import Role
+
+
+class Roles(object):
+    """
+    Client for interacting with OCS Roles
+    """
+
+    CommunityMember = 'f79a55da-7c76-4600-a809-0f62ca9971d9'
```

Comment:
Yes, most (probably all?) RoleTypeIds are hard-coded GUIDs. I could potentially expand this but generally didn't see the need to at this point.

Reply:
I think it would be better to include them all if they are hardcoded or to include none for consistency. It is not that important of a detail so I will approve your changes.

Reply:
is this used anywhere? I also think it might be good to name is specifically CommunityMemberRoleTypeId or something

Reply:
It's used inside the community steps themselves to identify which role is for the community member. 
I could remove it from the library and include it in the step itself so that it's more specific, probably easier than trying to enumerate all the guids.

Reply:
Something like this would be more helpful in the library than a sample, I think. To me it makes sense that the library would provide you with the actual guid you need if you need it in the sample, so I agree with the placement. My only real concern is that I can't tell that it's actually a RoleTypeId by looking at this section of code; it kind of just looks like a rogue role Id at first glance.

Reply:
I would think if there are other "well-known" guids that they should

Reply:
Fair enough :)

Reply:
Added all the standard Tenant and Community role type IDs here.